modules/distribution/src/repository/resources/conf/deployment.toml

```diff
@@ -9,11 +9,10 @@ password = "admin"
 create_admin_account = true
 
 [user_store]
-type = "read_write_ldap_unique_id"
-connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
-connection_name = "uid=admin,ou=system"
-connection_password = "admin"
-base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
+type = "database_unique_id"
+
+[embedded_ldap]
+enable = false
```

Comment:
are we not going to remove embedded ldap from the product?

Reply:
In this PR, we only fix the integration tests related issues. After finalizing the migration plan, we will remove embedded ldap from the product.